src/lib/ast.rs

```diff
@@ -364,10 +364,11 @@ fn parse_into_ast(pt: &parser::Node<u16>, grm: &Grammar, input: &str) -> Arena<S
         let (indent, e) = st.pop().unwrap();
         match e {
             &parser::Node::Terminal { lexeme } => {
-                let tid: usize = lexeme.tok_id().try_into().ok().unwrap();
-                let tn = grm.term_name(TIdx::from(tid)).unwrap();
-                let lt = &input[lexeme.start()..lexeme.start() + lexeme.len()];
-                child_node = arena.new_node(lt.to_string(), tn.to_string(), indent);
+                let token_id: usize = lexeme.tok_id().try_into().ok().unwrap();
+                let term_name = grammar.term_name(TIdx::from(token_id)).unwrap();
```

Comment:
`grm` is the convention in the `lr*` libraries too FWIW. It would be good to keep it the same here.